packages/blocks/header/src/app.tsx

```diff
@@ -1,10 +1,10 @@
 import { BlockComponent } from "@blockprotocol/graph/react";
-import { RefCallback } from "react";
+import { useHook, useHookBlockService } from "@blockprotocol/hook/react";
```

Comment:
You will need to ignore unresolved imports from the hook package as the eslint rule doesn't support subpath exports - see [this line](https://github.com/hashintel/hash/blob/2a7ca14837599a33c040b33a50cbee3e5fcf4dc4/.eslintrc.json#L98) – things will crash if they don't exist anyway / editors will tell us

Reply:
https://github.com/hashintel/hash/commit/d30863c01909221549152c43e49c66249e4e3f68#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090f